text/0193-removal-of-type-unions.md

```diff
@@ -0,0 +1,575 @@
+---
+feature name: removal-of-type-unions
+start date: 2020-07-06
+rfc pr: 194
+related issue: 193
+---
+
+# Summary
+
+_Type unions_ is a feature of the TypeScript language that allows typing values as "one of several candidate types".
+This is achieved by specifying a `|`-delimited list of candidate types (e.g: `Foo | Bar`). While this feature is common
+in dynamic languages (**TypeScript**, **Python**, ...), it is almost always absent from statically typed languages
+(**Java**, **C#**, ...). In those languages, the impossibility to represent _type unions_ natively results in degraded
+developer experience.
+
+This RFC proposes to remove all usage of _type unions_ from the AWS CDK, and eventually the removal of the feature from
+the `jsii` type model.
+
+# README
```

Comment:
Which README is this? Is this jsii readme? CDK?

Seems like this should probably be a "Release notes" type working backwards an not a README

Reply:
Yeah that does make sense.